lib/vnet/daemon/common_darwin.m

```diff
@@ -41,3 +48,73 @@
   }
   return strdup("");
 }
+
+bool getCodeSigningRequirement(NSString **outRequirement, NSError **outError) {
```

Comment:
This is a C function working on Obj-C objects, which I think is fine.

From what I understand, the pattern with passing stuff by reference is common when you want to "return" more than one thing. This is especially common when it comes to `NSError`. https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/ErrorHandling/ErrorHandling.html#//apple_ref/doc/uid/TP40011210-CH9-SW5